velox/functions/sparksql/DateTimeFunctions.h

```diff
@@ -807,6 +807,55 @@ struct MakeYMIntervalFunction {
   }
 };
 
+template <typename T>
+struct MakeDTIntervalFunction {
+  VELOX_DEFINE_FUNCTION_TYPES(T);
+
+  FOLLY_ALWAYS_INLINE void call(out_type<IntervalDayTime>& result) {
+    return call(result, 0, 0, 0, 0);
+  }
+
+  FOLLY_ALWAYS_INLINE void call(out_type<IntervalDayTime>& result, const int32_t& days) {
+    return call(result, days, 0, 0, 0);
+  }
+
+  FOLLY_ALWAYS_INLINE void call(out_type<IntervalDayTime>& result, const int32_t& days, const int32_t& hours) {
+    return call(result, days, hours, 0, 0);
+  }
+
+  FOLLY_ALWAYS_INLINE void call(out_type<IntervalDayTime>& result, const int32_t& days, const int32_t& hours, const int32_t& minutes) {
+    return call(result, days, hours, minutes, 0);
+  }
+
+  FOLLY_ALWAYS_INLINE void call(
+      out_type<IntervalDayTime>& result,
+      const int32_t days,
+      const int32_t hours,
+      const int32_t minutes,
+      const double seconds) {
```

Comment:
It looks like the fourth parameter is of short decimal type, so the input type should be int64_t, right?